CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ set(ROCMLIR_BIN_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin" CACHE PATH "")
 set(ROCMLIR_LIB_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib" CACHE PATH "")
 message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 
+set(MLIR_USE_CK OFF CACHE BOOL "Enable build CK")
```

Comment:
I don't quite like these names. Can we go for `ROCMLIR_ENABLE_BENCHMARKS` and then samething like `ROCMLIR_ENABLE_CK_BENCHMARKS`/`ROCMLIR_ENABLE_ROCBLAS_BENCHMARK` 

Or, alternatively, `ROCMLIR_ENABLE_BENCHMARKS=all` or `=ck;rocblas` or `=rocblas` or `=ck` or `=""` (to disable)

Reply:
I agree. I've made the changes and pushed them. But somehow this PR is not updated.

Reply:
Finally it got  updated.